Penteract/Assets/Scripts/HUDManager.cpp

```diff
@@ -177,17 +189,24 @@ void HUDManager::Update() {
 
 	ManageSwitch();
 	if (playingLostHealthFeedback) PlayLostHealthFeedback();
+	if (playingHitEffect) PlayHitEffect();
 }
 
 void HUDManager::UpdateCooldowns(float onimaruCooldown1, float onimaruCooldown2, float onimaruCooldown3, float fangCooldown1, float fangCooldown2, float fangCooldown3, float switchCooldown, float fangUltimateRemainingNormalizedValue, float oniUltimateRemainingNormalizedValue) {
 	cooldowns[static_cast<int>(Cooldowns::FANG_SKILL_1)] = fangCooldown1;
 	cooldowns[static_cast<int>(Cooldowns::FANG_SKILL_2)] = fangCooldown2;
-	cooldowns[static_cast<int>(Cooldowns::FANG_SKILL_3)] = fangUltimateRemainingNormalizedValue == 0 ? fangCooldown3 : 1.0f;
+	cooldowns[static_cast<int>(Cooldowns::FANG_SKILL_3)] = fangUltimateRemainingNormalizedValue == 0 ? fangCooldown3 : 0.99f;
```

Comment:
why is this a .99f?

Reply:
To solved an effect triggering bug with the abilities